packages/create-hint/src/create-hint.ts

```diff
@@ -190,7 +189,9 @@ class HintPackage {
  * ------------------------------------------------------------------------------
  */
 
-const mkdirpAsync = promisify(mkdirp) as (dir: string) => Promise<void>;
+const mkdirpAsync = (dir: string) => {
+    fs.mkdir(dir, {recursive: true});
```

Comment:
Don't you need an explicit `return` the way you've written this arrow function?

```suggestion
    return fs.mkdir(dir, {recursive: true});
```

Reply:
good catch.